config/locales/en.yml

```diff
@@ -81,6 +81,12 @@ en:
           attributes:
             team_id:
               same_team: "Inventory can't be shared to the same team as it belongs to"
+        my_module:
+          attributes:
+            x:
+             not_unique: "and Y position has already been taken by another task in the experiment."
```

Comment:
Shouldn't this be `X and Y` or does it append `X` automatically?

Reply:
X (attribute name) is appended automatically.